src/legacy/server/saved_objects/service/lib/decorate_es_error.ts

```diff
@@ -26,30 +26,33 @@ const {
   NoConnections,
   RequestTimeout,
   Conflict,
+  // @ts-ignore
   401: NotAuthorized,
+  // @ts-ignore
   403: Forbidden,
+  // @ts-ignore
   413: RequestEntityTooLarge,
   NotFound,
   BadRequest,
 } = elasticsearch.errors;
 
 import {
-  decorateBadRequestError,
-  decorateNotAuthorizedError,
-  decorateForbiddenError,
-  decorateRequestEntityTooLargeError,
   createGenericNotFoundError,
+  decorateBadRequestError,
   decorateConflictError,
   decorateEsUnavailableError,
+  decorateForbiddenError,
   decorateGeneralError,
+  decorateNotAuthorizedError,
+  decorateRequestEntityTooLargeError,
 } from './errors';
 
-export function decorateEsError(error) {
+export function decorateEsError(error: any) {
```

Comment:
Does it make sense to type `error: any` if the next line throws if it's not an `Error`?

Reply:
Yeah I agree, this should be types ad `Error`.